### ISSUE TITLE
Sensor Documentation

### ISSUE THREAD

**Issue by sambireddyt** [debrief/debrief#1717]
Provide documentation about wizarding used for Adding new sensor / sensor contact

http://www.debrief.info/tutorial/index.html#sensor_data.html

Debrief has two wizards, which are used to add a new sensor, or to add a new sensor cut to a sensor. As usual, for completeness, these should be in the reference section, probably at the above location.

Add new sensor:
![addsensor](https://cloud.githubusercontent.com/assets/1108513/15079312/aa438a5a-13af-11e6-9fb8-73ff40a3194e.png)

Add new sensor contact:
![addcontact](https://cloud.githubusercontent.com/assets/1108513/15079316/aff63862-13af-11e6-8979-20d6e11307e8.png)

Debrief has wizards for these data types because sometimes analysts receive sensor data as free text in an MS Word document. There may just be 1/2 dozen sensor cuts, so doing it "handraulically" is fine.  It's not an option to do this for positions, because there will always be so many.

**Comment by IanMayo** [comment]
Extra data, suitable for sensor wizards.

The status of the scenario is that we've added the two tracks, and some sensor data.

Let's say we've also got some hand-written notes from an exercise observer. The observer was able to record a periscope observation from Radar, so we have a really accurate range and bearing - so we're keen to include the data for when we conduct the plot-lock. We don't (yet) know how to produce the text-based REP files, and there are only a few cuts, so we're going to add them using the Sensor Wizard.

Sensor:
name:  EX_OBS_2
Color: Green

Sensor cuts:
1) Time:  03/Jan/1970 03:10:00 // Range: 27000 metres Bearing: 93.9 // Color: (keep as green)
2) Time:  03/Jan/1970 03:25:00 // Range: 17150 metres Bearing: 85.4 // Color: (keep as green)

When we adjust the stacked dots, we'll see these green markers near the bottom of the display (earlier in the engagement) - and we can be sure to acknowledge them when we're minimising the errors.
